pulse_actions/handlers/treeherder_buildbot.py

```diff
@@ -21,7 +21,14 @@ def on_buildbot_event(data, message, dry_run):
     treeherder_client = TreeherderClient()
     repo_name = data['project']
     job_id = data['job_id']
-    result = treeherder_client.get_jobs(repo_name, id=job_id)[0]
+    result = treeherder_client.get_jobs(repo_name, id=job_id)
+    # If result not found, ignore
+    if not result:
+        LOG.info("We could not find any result for repo_name: %s and job_id: %s" % (repo_name, job_id))
+        message.ack()
+        return
```

Comment:
I noticed yesterday that some random job_ids were getting passed which did not exists, and that cause out of index errors, we just would want to ignore such requests, and log them (for further intervention)